tests/README.md

```diff
@@ -22,15 +22,21 @@ Validation tests have the same requirements as integration tests however differe
 
 ### Integration vs Validation
 
-TODO
+Integration tests - Don't require any external configuration or access to any other external services. These should also be short running as they will run on every PR within CI. Integration tests do not need a configuration file to run.
+
+Validation tests - Require access to external services, and needs to a config file to run them.
```

Comment:
I would call out that it is not a config file but more specifically access keys to cloud providers.